lib/MusicBrainz/Server/Data/EditNote.pm

```diff
@@ -120,9 +120,9 @@ sub find_by_recipient {
 
     my $query = <<EOSQL;
         SELECT ${\($self->_columns)}
-          FROM ${\($self->_table)}
+          FROM edit_note_recipient
+          JOIN ${\($self->_table)} ON ${\($self->_table)}.id = edit_note_recipient.edit_note_id
          WHERE editor != \$1
-           AND edit IN (SELECT id FROM edit WHERE editor = \$1)
            AND post_time > now() - interval '3 months'
```

Comment:
I think we can remove this 3-month limitation now, it should be fast enough if you wanna view edit notes further back than that.